resources/js/components/Elements/Slider.vue

```diff
@@ -1,71 +1,115 @@
 <script>
-    export default {
-        render() {
-            return this.$scopedSlots.default({
-                navigate: this.navigate,
-                showLeft: this.showLeft,
-                showRight: this.showRight,
-                currentSlide: this.currentSlide,
-                slidesTotal: this.slidesTotal
-            })
+export default {
+    render() {
+        return this.$scopedSlots.default({
+            navigate: this.navigate,
+            showLeft: this.showLeft,
+            showRight: this.showRight,
+            currentSlide: this.currentSlide,
+            slidesTotal: this.slidesTotal,
+        })
+    },
+    props: {
+        reference: {
+            type: String,
+            default: 'slider',
         },
-        props: {
-            reference: {
-                type: String,
-                default: 'slider'
-            },
-            vertical: {
-                type: Boolean,
-                default: false
-            }
+        vertical: {
+            type: Boolean,
+            default: false,
         },
-        data: () => {
-            return {
-                position: 0,
-                showLeft: false,
-                showRight: false,
-                mounted: false
-            }
+        interval: {
```

Comment:
Is this the best naming? Maybe autoScrollInterval? Or maybe better; have 2 props: autoscroll, bool and default false and just interval like this with a proper default?